### PR TITLE
N°6419 - Update hungarian translations thanks to @tacsaby

### PR DESCRIPTION
**Pull request by tacsaby**
Some corrections

### CODE REVIEW

**Comment by tacsaby** [comment]
Please update translation by recent changes! Thanks!

**Comment by piRGoif** [comment]
Hello,
I see you emptied lots of "+" dict entries. Can you explain why ? 
Also lots of apos were removed around parameters, for example on `Class:User/Error:LoginMustBeUnique`. They are present in other languages, why should they be removed in HU ?

**Comment by piRGoif** [comment]
Hello @tacsaby , any news ?

**Comment by tacsaby** [comment]
Hello,

I removed some "+" dict entries because it was trivial in my language,
makes not sense to describe it more.
Don't know why, but if parameters in " double quotes, the values of
parameter not printed, but the name of parameter.  This is not desirable.
I wait a little with sending all my translations, if the new translation
app released.
I have problems to commit translation on github, because that not familiar
to me and you don't like the method that i can do that.


Pierre Goiffon ***@***.***> ezt írta (időpont: 2023. jún. 6.,
K 15:20):

File: dictionaries/hu.dictionary.itop.ui.php [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-    'Class:AuditCategory/Attribute:name' => 'Kategórinév',
+    'Class:AuditCategory/Attribute:name' => 'Kategórianév',
     'Class:AuditCategory/Attribute:name+' => '',
     'Class:AuditCategory/Attribute:description' => 'Leírás',
     'Class:AuditCategory/Attribute:description+' => '',
     'Class:AuditCategory/Attribute:definition_set' => 'Definíciókészlet',
     'Class:AuditCategory/Attribute:definition_set+' => '',
     'Class:AuditCategory/Attribute:rules_list' => 'Auditszabályok',
-    'Class:AuditCategory/Attribute:rules_list+' => 'Audit rules for this category~~',
+    'Class:AuditCategory/Attribute:rules_list+' => '',
```
Comment (Hipska) [inline dictionaries/hu.dictionary.itop.ui.php]: So you are saying it does not make sense to have a description (or even a translated description) for this list in your language?

File: dictionaries/hu.dictionary.itop.ui.php [hunk 2]
```diff
@@ -100,17 +100,17 @@
...
-    'Class:User/Attribute:status+' => 'Whether the user account is enabled or disabled.~~',
+    'Class:User/Attribute:status+' => '',
     'Class:User/Attribute:status/Value:enabled' => 'Engedélyezett',
     'Class:User/Attribute:status/Value:disabled' => 'Letiltott',
 
-    'Class:User/Error:LoginMustBeUnique' => 'A felhasználónévnek egyedinek kell lennie - "%1s" már létezik.',
+    'Class:User/Error:LoginMustBeUnique' => 'A felhasználónévnek egyedinek kell lennie - %1s már létezik.',
     'Class:User/Error:AtLeastOneProfileIsNeeded' => 'Legalább egy profilt a felhasználóhoz kell rendelni.',
-    'Class:User/Error:ProfileNotAllowed' => 'A "%1$s" profil nem adható hozzá, le lesz tiltva',
+    'Class:User/Error:ProfileNotAllowed' => 'A %1$s profil nem adható hozzá, le lesz tiltva',
```
Comment (Hipska) [inline dictionaries/hu.dictionary.itop.ui.php]: Double quotes in single quotes should just work. Please revert this change. (and all others)

**Review by Hipska — CHANGES_REQUESTED** [review]

File: dictionaries/hu.dictionary.itop.ui.php (hunk 1 above)
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:AuditCategory/Attribute:rules_list+' => 'Audit rules for this category~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 3]
```diff
@@ -50,19 +50,19 @@
 
 Dict::Add('HU HU', 'Hungarian', 'Magyar', array(
     'Class:Query' => 'Lekérdezés',
-    'Class:Query+' => 'A query is a data set defined in a dynamic way~~',
+    'Class:Query+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:Query+' => 'A query is a data set defined in a dynamic way~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 4]
```diff
@@ -50,19 +50,19 @@
...
     'Class:Query/Attribute:name' => 'Név',
-    'Class:Query/Attribute:name+' => 'Identifies the query~~',
+    'Class:Query/Attribute:name+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:Query/Attribute:name+' => 'Identifies the query~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 5]
```diff
@@ -50,19 +50,19 @@
...
     'Class:Query/Attribute:description' => 'Leírás',
-    'Class:Query/Attribute:description+' => 'Long description for the query (purpose, usage, etc.)~~',
+    'Class:Query/Attribute:description+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:Query/Attribute:description+' => 'Long description for the query (purpose, usage, etc.)~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 6]
```diff
@@ -50,19 +50,19 @@
...
     'Class:Query/Attribute:is_template' => 'OQL mező sablonok',
-    'Class:Query/Attribute:is_template+' => 'Usable as source for recipient OQL in Notifications~~',
+    'Class:Query/Attribute:is_template+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:Query/Attribute:is_template+' => 'Usable as source for recipient OQL in Notifications~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 7]
```diff
@@ -50,19 +50,19 @@
...
     'Class:Query/Attribute:is_template/Value:yes' => 'Igen',
     'Class:Query/Attribute:is_template/Value:no' => 'Nem',
     'Class:QueryOQL/Attribute:fields' => 'Mezők',
-    'Class:QueryOQL/Attribute:fields+' => 'Comma separated list of attributes (or alias.attribute) to export~~',
+    'Class:QueryOQL/Attribute:fields+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:QueryOQL/Attribute:fields+' => 'Comma separated list of attributes (or alias.attribute) to export~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 8]
```diff
@@ -50,19 +50,19 @@
...
     'Class:QueryOQL' => 'OQL lekérdezés',
-    'Class:QueryOQL+' => 'A query based on the Object Query Language~~',
+    'Class:QueryOQL+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:QueryOQL+' => 'A query based on the Object Query Language~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 9]
```diff
@@ -100,17 +100,17 @@
     'Class:User/Attribute:language/Value:FR FR' => 'Francia',
     'Class:User/Attribute:language/Value:FR FR+' => '',
     'Class:User/Attribute:profile_list' => 'Profil',
-    'Class:User/Attribute:profile_list+' => 'Roles, granting rights for that person~~',
+    'Class:User/Attribute:profile_list+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:User/Attribute:profile_list+' => 'Roles, granting rights for that person~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 10]
```diff
@@ -100,17 +100,17 @@
...
     'Class:User/Attribute:allowed_org_list' => 'Engedélyezett szervezeti egységek',
-    'Class:User/Attribute:allowed_org_list+' => 'The end user is allowed to see data belonging to the following organizations. If no organization is specified, there is no restriction.~~',
+    'Class:User/Attribute:allowed_org_list+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:User/Attribute:allowed_org_list+' => 'The end user is allowed to see data belonging to the following organizations. If no organization is specified, there is no restriction.~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 11]
```diff
@@ -100,17 +100,17 @@
...
     'Class:User/Attribute:status' => 'Állapot',
-    'Class:User/Attribute:status+' => 'Whether the user account is enabled or disabled.~~',
+    'Class:User/Attribute:status+' => '',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:User/Attribute:status+' => 'Whether the user account is enabled or disabled.~~',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 12]
```diff
@@ -100,17 +100,17 @@
...
     'Class:User/Attribute:status/Value:enabled' => 'Engedélyezett',
     'Class:User/Attribute:status/Value:disabled' => 'Letiltott',
 
-    'Class:User/Error:LoginMustBeUnique' => 'A felhasználónévnek egyedinek kell lennie - "%1s" már létezik.',
+    'Class:User/Error:LoginMustBeUnique' => 'A felhasználónévnek egyedinek kell lennie - %1s már létezik.',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:User/Error:LoginMustBeUnique' => 'A felhasználónévnek egyedinek kell lennie - "%1s" már létezik.',
```

File: dictionaries/hu.dictionary.itop.ui.php (hunk 2 above)
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'Class:User/Error:ProfileNotAllowed' => 'A "%1$s" profil nem adható hozzá, le lesz tiltva',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 13]
```diff
@@ -424,27 +424,27 @@
...
     'UI:Error:SaveFailed' => 'Az objektum nem menthető le :',
-    'UI:Error:MandatoryTemplateParameter_object_id' => 'object_id pareméter kötelező a link_attr megadásánál. Ellenőrizze a sablon definíciót.',
-    'UI:Error:MandatoryTemplateParameter_target_attr' => 'target_attr pareméter kötelező a link_attr megadásánál. Ellenőrizze a sablon definíciót.',
+    'UI:Error:MandatoryTemplateParameter_object_id' => 'object_id paraméter kötelező a link_attr megadásánál. Ellenőrizze a sablon definíciót.',
+    'UI:Error:MandatoryTemplateParameter_target_attr' => 'target_attr paraméter kötelező a link_attr megadásánál. Ellenőrizze a sablon definíciót.',
     'UI:Error:MandatoryTemplateParameter_group_by' => 'group_by paraméter kötelező. Ellenőrizze a sablon definíciót.',
-    'UI:Error:InvalidGroupByFields' => 'Csoportosításnál használt érvénytelen mezők: "%1$s".',
-    'UI:Error:UnsupportedStyleOfBlock' => 'Hiba: nem támogatott stílus tömb: "%1$s".',
+    'UI:Error:InvalidGroupByFields' => 'Csoportosításnál használt érvénytelen mezők: %1$s.',
+    'UI:Error:UnsupportedStyleOfBlock' => 'Hiba: nem támogatott stílus tömb: %1$s.',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Error:InvalidGroupByFields' => 'Csoportosításnál használt érvénytelen mezők: "%1$s".',
    'UI:Error:UnsupportedStyleOfBlock' => 'Hiba: nem támogatott stílus tömb: "%1$s".',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 14]
```diff
@@ -424,27 +424,27 @@
...
+    'UI:Error:WizardCircularReferenceInDependencies' => 'Hiba: Körkörös hivatkozás az egymásra mutató mezők között. Ellenőrizze az adatmodellt.',
+    'UI:Error:UploadedFileTooBig' => 'Feltöltendő fájl túl nagy. (Maximális méret: %1$s). Ellenőrizze a PHP konfigurációs fájlban az upload_max_filesize és post_max_size beállításokat.',
     'UI:Error:UploadedFileTruncated.' => 'Feltöltött fájl átméretezett!',
     'UI:Error:NoTmpDir' => 'Az átmeneti könyvtár nem meghatározott.',
-    'UI:Error:CannotWriteToTmp_Dir' => 'Az átmeneti fájl nem írható. upload_tmp_dir = "%1$s".',
-    'UI:Error:UploadStoppedByExtension_FileName' => 'Feltöltés megállt a fájl kiterjesztés miatt. (Eredeti fájl név = "%1$s").',
-    'UI:Error:UploadFailedUnknownCause_Code' => 'Fájl feltöltés sikertelen ismeretlen hiba miatt. (Hibakód = "%1$s").',
+    'UI:Error:CannotWriteToTmp_Dir' => 'Az átmeneti fájl nem írható. upload_tmp_dir = %1$s.',
+    'UI:Error:UploadStoppedByExtension_FileName' => 'Feltöltés megállt a fájl kiterjesztés miatt. (Eredeti fájl név = %1$s).',
+    'UI:Error:UploadFailedUnknownCause_Code' => 'Fájl feltöltés sikertelen ismeretlen hiba miatt. (Hibakód = %1$s).',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Error:CannotWriteToTmp_Dir' => 'Az átmeneti fájl nem írható. upload_tmp_dir = "%1$s".',
    'UI:Error:UploadStoppedByExtension_FileName' => 'Feltöltés megállt a fájl kiterjesztés miatt. (Eredeti fájl név = "%1$s").',
    'UI:Error:UploadFailedUnknownCause_Code' => 'Fájl feltöltés sikertelen ismeretlen hiba miatt. (Hibakód = "%1$s").',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 15]
```diff
@@ -454,7 +454,7 @@
     'UI:Error:BulkModifyNotAllowedOn_Class' => 'Az osztály objektumainak tömeges frissítése nem engedélyezett %1$s',
     'UI:Error:ObjectAlreadyCloned' => 'Hiba: az objektum már klónozott!',
     'UI:Error:ObjectAlreadyCreated' => 'Hiba: az objektum már létrehozva!',
-    'UI:Error:Invalid_Stimulus_On_Object_In_State' => 'Hiba: érvénytelen stimulus "%1$s" a következő objektum %2$s következő állapotában "%3$s".',
+    'UI:Error:Invalid_Stimulus_On_Object_In_State' => 'Hiba: érvénytelen stimulus %1$s a következő objektum %2$s következő állapotában %3$s.',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Error:Invalid_Stimulus_On_Object_In_State' => 'Hiba: érvénytelen stimulus "%1$s" a következő objektum %2$s következő állapotában "%3$s".',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 16]
```diff
@@ -532,7 +532,7 @@
     'UI:Login:ResetPwdFailed' => 'Sikertelen email küldés: %1$s',
     'UI:Login:SeparatorOr' => 'Vagy',
 
-    'UI:ResetPwd-Error-WrongLogin' => '\'%1$s\' nem érvényes fiók',
+    'UI:ResetPwd-Error-WrongLogin' => '%1$s nem érvényes fiók',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:ResetPwd-Error-WrongLogin' => '\'%1$s\' nem érvényes fiók',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 17]
```diff
@@ -578,21 +578,21 @@
...
     'UI:CSVImport:DataLine1'                       => 'Adatsor 1',
     'UI:CSVImport:DataLine2'                       => 'Adatsor 2',
-    'UI:CSVImport:idField'                         => 'id (elsődeges kulcs)',
+    'UI:CSVImport:idField'                         => 'id (elsődleges kulcs)',
     'UI:Title:BulkImport' => ITOP_APPLICATION_SHORT.' - tömeges betöltés',
     'UI:Title:BulkImport+'                         => '',
     'UI:Title:BulkSynchro_nbItem_ofClass_class'    => '%2$s osztály %1$d objektumának szinkronizációja',
     'UI:CSVImport:ClassesSelectOne'                => '-- válasszon ki egyet --',
-    'UI:CSVImport:ErrorExtendedAttCode'            => 'Belső hiba: "%1$s" nem megfelelő kód, mert "%2$s" nem külső kulcsa a "%3$s" osztálynak',
+    'UI:CSVImport:ErrorExtendedAttCode'            => 'Belső hiba: %1$s nem megfelelő kód, mert %2$s nem külső kulcsa a %3$s osztálynak',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:CSVImport:ErrorExtendedAttCode'            => 'Belső hiba: "%1$s" nem megfelelő kód, mert "%2$s" nem külső kulcsa a "%3$s" osztálynak',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 18]
```diff
@@ -634,8 +634,8 @@
 
     'UI:CSVReport-Value-Modified' => 'Módosítva',
     'UI:CSVReport-Value-SetIssue' => 'Érvénytelen érték az attribútumhoz',
-    'UI:CSVReport-Value-ChangeIssue' => '\'%1$s\' egy érvénytelen érték',
-    'UI:CSVReport-Value-NoMatch' => 'Nincs egyezés a \'%1$s\' értékhez',
+    'UI:CSVReport-Value-ChangeIssue' => '%1$s egy érvénytelen érték',
+    'UI:CSVReport-Value-NoMatch' => 'Nincs egyezés a %1$s értékhez',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:CSVReport-Value-ChangeIssue' => '\'%1$s\' egy érvénytelen érték',
    'UI:CSVReport-Value-NoMatch' => 'Nincs egyezés a \'%1$s\' értékhez',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 19]
```diff
@@ -646,10 +646,10 @@
     'UI:CSVReport-Value-Issue-Null' => 'A nulla nem engedélyezett',
     'UI:CSVReport-Value-Issue-NotFound' => 'Az objektum nincs meg',
     'UI:CSVReport-Value-Issue-FoundMany' => '%1$d egyezés található',
-    'UI:CSVReport-Value-Issue-Readonly' => 'A \'%1$s\' attribútum csak olvasható (jelenlegi érték: %2$s, várható érték: %3$s)',
+    'UI:CSVReport-Value-Issue-Readonly' => 'A %1$s attribútum csak olvasható (jelenlegi érték: %2$s, várható érték: %3$s)',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:CSVReport-Value-Issue-Readonly' => 'A \'%1$\'s attribútum csak olvasható (jelenlegi érték: %2$s, várható érték: %3$s)',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 20]
```diff
@@ -646,10 +646,10 @@
...
     'UI:CSVReport-Value-Issue-Format' => 'A bevitel feldolgozása sikertelen: %1$s',
-    'UI:CSVReport-Value-Issue-NoMatch' => 'A \'%1$s\' attribútum nem várt értéket kapott: nincs egyezés, ellenőrizze a beírást',
-    'UI:CSVReport-Value-Issue-Unknown' => 'A \'%1$s\' attribútum nem várt értéket kapott: %2$s',
+    'UI:CSVReport-Value-Issue-NoMatch' => 'A %1$s attribútum nem várt értéket kapott: nincs egyezés, ellenőrizze a beírást',
+    'UI:CSVReport-Value-Issue-Unknown' => 'A %1$s attribútum nem várt értéket kapott: %2$s',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:CSVReport-Value-Issue-NoMatch' => 'A \'%1$s\' attribútum nem várt értéket kapott: nincs egyezés, ellenőrizze a beírást',
    'UI:CSVReport-Value-Issue-Unknown' => 'A \'%1$s\' attribútum nem várt értéket kapott: %2$s',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 21]
```diff
@@ -723,10 +723,10 @@
     'UI:Schema:AbstractClass' => 'Absztrakt osztály: nem példányosítható belőle objektum.',
     'UI:Schema:NonAbstractClass' => 'Nem absztrakt osztály: objektum példányosítható belőle.',
     'UI:Schema:ClassHierarchyTitle' => 'Osztály hierarchia',
-    'UI:Schema:AllClasses' => 'Összes osztály',
+    'UI:Schema:AllClasses' => 'Osztályok',
     'UI:Schema:ExternalKey_To' => 'Külső kulcs %1$s-hoz',
     'UI:Schema:Columns_Description' => 'Oszlopok: <em>%1$s</em>',
-    'UI:Schema:Default_Description' => 'Alapértelmezett: "%1$s"',
+    'UI:Schema:Default_Description' => 'Alapértelmezett: %1$s',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Schema:Default_Description' => 'Alapértelmezett: "%1$s"',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 22]
```diff
@@ -782,7 +782,7 @@
     'UI:Schema:DisplaySelector/Label' => 'Címke',
     'UI:Schema:DisplaySelector/Code' => 'Kód',
     'UI:Schema:Attribute/Filter' => 'Szűrő',
-    'UI:Schema:DefaultNullValue' => 'Alapértelmezett null érték : "%1$s"',
+    'UI:Schema:DefaultNullValue' => 'Alapértelmezett null érték : %1$s',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Schema:DefaultNullValue' => 'Alapértelmezett null érték : "%1$s"',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 23]
```diff
@@ -849,10 +849,10 @@
     'UI:SearchResultsTitle' => 'Keresés eredményei',
     'UI:SearchResultsTitle+' => 'Szöveges keresés eredményei',
     'UI:Search:NoSearch' => 'Nincs keresés',
-    'UI:Search:NeedleTooShort' => 'A \\"%1$s\\" kereső karakterlánc túl rövid. Legalább %2$d karaktert írjon be.',
-    'UI:Search:Ongoing' => '\\"%1$s\\" keresése',
+    'UI:Search:NeedleTooShort' => 'A %1$s kereső karakterlánc túl rövid. Legalább %2$d karaktert írjon be.',
+    'UI:Search:Ongoing' => '%1$s keresése',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:Search:NeedleTooShort' => 'A \\"%1$s\\" kereső karakterlánc túl rövid. Legalább %2$d karaktert írjon be.',
    'UI:Search:Ongoing' => '\\"%1$s\\" keresése',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 24]
```diff
@@ -849,10 +849,10 @@
...
     'UI:Search:Enlarge' => 'Keresés kibővítése',
-    'UI:FullTextSearchTitle_Text' => '"%1$s" keresés eredményei:',
+    'UI:FullTextSearchTitle_Text' => '%1$s keresés eredményei:',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:FullTextSearchTitle_Text' => '"%1$s" keresés eredményei:',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 25]
```diff
@@ -1203,8 +1203,8 @@
     'UI:Button:MoveUp' => 'Feljebb',
     'UI:Button:MoveDown' => 'Lejjebb',
 
-    'UI:OQL:UnknownClassAndFix' => 'Ismeretlen osztály \\"%1$s\\". Próbálja meg \\"%2$s\\" -t helyette',
-    'UI:OQL:UnknownClassNoFix' => 'Ismeretlen osztály \\"%1$s\\"',
+    'UI:OQL:UnknownClassAndFix' => 'Ismeretlen osztály: %1$s. Próbálja meg %2$s -t helyette',
+    'UI:OQL:UnknownClassNoFix' => 'Ismeretlen osztály: %1$s',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:OQL:UnknownClassAndFix' => 'Ismeretlen osztály: \\"%1$s\\". Próbálja meg \\"%2$s\\" -t helyette',
    'UI:OQL:UnknownClassNoFix' => 'Ismeretlen osztály: \\"%1$s\\"',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 26]
```diff
@@ -1242,13 +1242,13 @@
     'UI:DashletUnknown:Label' => 'Ismeretlen',
     'UI:DashletUnknown:Description' => 'Ismeretlen műszer (talán eltávolították)',
     'UI:DashletUnknown:RenderText:View' => 'Nem lehet megjeleníteni ezt a műszert.',
-    'UI:DashletUnknown:RenderText:Edit' => 'Nem lehet megjeleníteni ezt a műszert ("%1$s" osztály). Ellenőriztesse a rendszergazdával, hogy elérhető-e.',
-    'UI:DashletUnknown:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a műszerhez ("%1$s" osztály).',
+    'UI:DashletUnknown:RenderText:Edit' => 'Nem lehet megjeleníteni ezt a műszert (%1$s osztály). Ellenőriztesse a rendszergazdával, hogy elérhető-e.',
+    'UI:DashletUnknown:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a műszerhez (%1$s osztály).',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:DashletUnknown:RenderText:Edit' => 'Nem lehet megjeleníteni ezt a műszert ("%1$s" osztály). Ellenőriztesse a rendszergazdával, hogy elérhető-e.',
    'UI:DashletUnknown:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a műszerhez ("%1$s" osztály).',
```

File: dictionaries/hu.dictionary.itop.ui.php [hunk 27]
```diff
@@ -1242,13 +1242,13 @@
...
     'UI:DashletUnknown:Prop-XMLConfiguration' => 'Konfiguráció (nyers XML)',
 
     'UI:DashletProxy:Label' => 'Proxy',
     'UI:DashletProxy:Description' => 'Proxy műszer',
-    'UI:DashletProxy:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a harmadik féltől származó műszerhez ("%1$s" osztály).',
+    'UI:DashletProxy:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a harmadik féltől származó műszerhez (%1$s osztály).',
```
Comment (Molkobain) [inline dictionaries/hu.dictionary.itop.ui.php]:
```suggestion
    'UI:DashletProxy:RenderNoDataText:Edit' => 'Nincs előnézet ehhez a harmadik féltől származó műszerhez ("%1$s" osztály).',
```

**Review by Molkobain — CHANGES_REQUESTED** [review]

**Review by Molkobain — APPROVED** [review]

**Review by Hipska — APPROVED** [review]

**Comment by Molkobain** [comment]
Accepted during functional review

**Pull request merged**